src/app/modules/group/components/suggestion-of-activities/suggestion-of-activities.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { ItemNavigationService, RootActivity } from '../../../../core/http-services/item-navigation.service';
+import { UserSessionService } from '../../../../shared/services/user-session.service';
+import { switchMap, filter, map } from 'rxjs/operators';
+import { isNotUndefined } from '../../../../shared/helpers/null-undefined-predicates';
+import { mapToFetchState } from '../../../../shared/operators/state';
+
+@Component({
+  selector: 'alg-suggestion-of-activities',
+  templateUrl: './suggestion-of-activities.component.html',
+  styleUrls: [ './suggestion-of-activities.component.scss' ],
+})
+export class SuggestionOfActivitiesComponent {
+  readonly state$ = this.sessionService.watchedGroup$.pipe(
```

Comment:
Nice way to retrieve the id of clicked group without strange ways of passing the parameters! 👍